src/icon/README.md

```diff
@@ -0,0 +1,62 @@
+# @dojo/widgets/icon/Icon widget
+
+Dojo 2's `Icon` widget renders an icon.
+
+
+## Example Usage
+
+*Basic Example*
+```typescript
+import Icon from '@dojo/widgets/icon/Icon';
+import { w } from '@dojo/widget-core/d';
+
+w(Icon, { type: 'downIcon', });
+```
+
+By default the icon will be rendered with `role="presentation"` and
+`aria-hidden="true"`. Custom aria attributes can be specified to
```

Comment:
Really quick word update to something like "The icon will be rendered with `role="presentation"` and
 +`aria-hidden="true"` if `altText` is not provided"

Reply:
Updated this, but with slightly different wording since we'll always have `aria-hidden="true"` regardless of whether `altText` is provided.